dockstore-client/src/test/java/io/dockstore/client/cli/MissingOutputParasTest.java

```diff
@@ -0,0 +1,107 @@
+package io.dockstore.client.cli.nested;
+
+import org.apache.commons.io.FileUtils;
+import org.junit.After;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.contrib.java.lang.system.ExpectedSystemExit;
+import org.junit.runner.RunWith;
+import org.powermock.modules.junit4.PowerMockRunner;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.PrintWriter;
+import java.util.List;
+
+import static org.junit.Assert.assertTrue;
+
+/**
+ * Created by kcao on 17/02/17.
+ */
+
+public class MissingOutputParasTest {
+    @Rule
+    public final ExpectedSystemExit exit = ExpectedSystemExit.none();
+
+    public static final AbstractEntryClient entryClient = new ToolClient(null, false);
+    public static File cwlTestFile;
+    public static StringBuilder stringBuilder = new StringBuilder("cwlVersion: cwl:draft-3\n");
+
+    @BeforeClass
+    public static void baseCWL() {
+        cwlTestFile = new File(System.getProperty("user.home") + File.separator + "missingOutputParas.cwl");
```

Comment:
Thanks for the testing, it looks like a good start. 
Same comment here though, store your test files in properties, don't create them in your user's home directory. 
(ex: https://github.com/ga4gh/dockstore/blob/8ef1940bb24897f9f1a749506abb308b5a6c5700/dockstore-client/src/test/java/io/dockstore/client/cli/LaunchTestIT.java#L209 )